siac/walletcmd.go

```diff
@@ -61,7 +61,7 @@ By default the wallet encryption / unlock password is the same as the generated
 	}
 
 	walletLoad033xCmd = &cobra.Command{
-		Use:   "033x [filepath]",
+		Use:   "033x",
```

Comment:
If you remove `[filepath]` here, consider also removing `[filepaths]` from `load siag`. Or inversely, consider keeping `[filepath]` here.